### PR TITLE
Cleaned up docker files and added minor version tag

### PR DESCRIPTION
**Pull request by wsobel**
* Cleaned up docker files per comments.

### CODE REVIEW

File: .github/workflows/build-docker-image.yml [hunk 1]
```diff
@@ -68,6 +68,7 @@ jobs:
       - name: Remove leading v from the version
         run: |
           echo "VERSION=$(echo ${{ github.ref_name }} | sed 's/^v//')" >> $GITHUB_ENV
```
Comment (ottobolyos) [inline .github/workflows/build-docker-image.yml:70]: Useless use of `echo`.

```suggestion
          echo "VERSION='$(sed 's/^v//' <<< ${{github.ref_name}})'" >> $GITHUB_ENV
```

File: .github/workflows/build-docker-image.yml [hunk 2]
```diff
@@ -68,6 +68,7 @@ jobs:
...
+          echo "MAJOR_VERSION=$(echo ${{ github.ref_name }} | sed -E 's/^v([0-9]+)\.([0-9]+)\..+/\1.\2/')" >> $GITHUB_ENV
```
Comment (ottobolyos) [inline .github/workflows/build-docker-image.yml:71]: Useless use of `echo`. And actually, it is not a _major_ version you get, but major and minor version (or simply a minor version), OFC only if you follow semantic versioning.

```suggestion
          echo "MAJOR_VERSION=$(sed -E 's/^v([0-9]+\.[0-9]+).*/\1/')" <<< ${{github.ref_name}} >> $GITHUB_ENV
```

Or without `-E`:

```suggestion
          echo "MAJOR_VERSION=$(sed 's/^v\([0-9]\+\.[0-9]\+\).*/\1/')" <<< ${{github.ref_name}} >> $GITHUB_ENV
```

**Review by ottobolyos — CHANGES_REQUESTED** [review]

File: .github/workflows/build-docker-image.yml (hunk 1 above)
Comment (wsobel) [reply .github/workflows/build-docker-image.yml:70]: This is the pattern used in the GitHub workflow documentation. I don't think it is any less easy to read or understand than binding stdin. 

I find the pipe clearer to understand the flow of data than the way you wrote it.

File: .github/workflows/build-docker-image.yml (hunk 2 above)
Comment (wsobel) [reply .github/workflows/build-docker-image.yml:71]: I could simplify. I'm not concerned about the major version name. Went with a few terms and then decided major and minor was too wordy.

**Pull request merged**